CHANGELOG.md

```diff
@@ -1,5 +1,99 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.11.12
+
+<!-- generated comparing v5.11.11..master -->
+
+_Mar 6, 2023_
+
+A big thanks to the 17 contributors who made this release possible. Here are some highlights ✨:
+
+- @michaldudak added the multiselect functionality to SelectUnstyled (#36274)
+- other 🐛 bug fixes and 📚 documentation improvements.
```

Comment:
We can add https://github.com/mui/material-ui/pull/35739 in the highlights - it's about updating `extendTheme` to generate CSS vars with default values - which makes the CssVarsProvider no longer required for Joy UI (for the default styles)